addons/sys_intercom/fnc_updateInfantryPhoneStatus.sqf

```diff
@@ -60,5 +60,5 @@ switch (_action) do {
 };
 
 // Hook for third party mods with actions when picking returning infantry phone
-private _event = _vehicle getVariable [QGVAR(eventInfantryPhone), FUNC(noApiFunction)];
-[_vehicle, _unit, _action] call (missionNamespace getVariable [_event, FUNC(noApiFunction)]);
\ No newline at end of file
+private _event = _vehicle getVariable [QGVAR(eventInfantryPhone), {}];
+[_vehicle, _unit, _action] call (missionNamespace getVariable [_event, {}]);
```

Comment:
This could be:
```
private _event = _vehicle getVariable [QGVAR(eventInfantryPhone), ""];
if (_event != "") then {
    _event = missionNamespace getVariable [_event, {}];
    if !(_event isEqualTo {}) then {
        [_vehicle, _unit, _action] call _event;
    };
};
```
for as little overhead as possible.